data-dashboard/lib/components/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+import React from 'react';
+
+export interface TextInputProps {
```

Comment:
Exporting the props of a component is a sign of tight coupling between components.
```suggestion
interface TextInputProps {
```

Reply:
I don't get why this would be bad, I will make some research.